.github/workflows/minikube-zap-test.yml

```diff
@@ -0,0 +1,43 @@
+# This is a basic workflow to help you get started with Actions
+
+name: DAST with ZAP on minikube
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches: [master]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  issues: write
```

Comment:
???? why write?

Reply:
Thanks! Can we instead just file the results as part of the job?

Reply:
Done